models.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import BaseClass
```

Comment:
The import statement should be `from database import BaseClass as Base` to align with the typical convention of using `Base` as the declarative base class name. This will also match the usage in the Alembic `env.py` file where `BaseClass` is referenced as `Base`.